pytatki/user.py

```diff
@@ -194,6 +194,59 @@ def login_get():
     return render_template('login.html')
 
 
+@APP.route('/resetpassword/', methods=["GET"])
```

Comment:
Słowa w url'u powinny być oddzielone myślnikiem
```suggestion
@APP.route('/reset-password/', methods=["GET"])
```